WordPress/Classes/Networking/SharingServiceRemote.swift

```diff
@@ -40,23 +43,22 @@ public class SharingServiceRemote : ServiceRemoteREST
     ///
     public func getPublicizeServices(success: ([RemotePublicizeService] -> Void)?, failure: (NSError! -> Void)?) {
         let endpoint = "meta/external-services"
-        let path = pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
-        let params = NSDictionary(object: "publicize", forKey: "type")
+        let path = pathForEndpoint(endpoint, withVersion: .Version_1_1)
+        let params = ["type":"publicize"]
 
-        api.GET(path,
+        wordPressComRestApi.GET(path,
             parameters: params,
-            success: { (operation: AFHTTPRequestOperation!, response: AnyObject!) in
+            success: { (responseObject: AnyObject, httpResponse: NSHTTPURLResponse?) in
                 guard let onSuccess = success else {
                     return
                 }
 
-                // For paranioa, make sure the response is the correct type.
-                guard let responseDict = response as? NSDictionary else {
-                    failure?(self.errorForUnexpectedResponse(operation))
+                guard let responseDict = responseObject as? NSDictionary else {
+                    failure?(self.errorForUnexpectedResponse(httpResponse))
                     return
                 }
 
-                let responseString = operation.responseString! as NSString
+                let responseString = responseObject.description as NSString
```

Comment:
Hummmm, this is going to be problematic, the new API interface based on the classes of AFNetworking for NSURLSession don't provide the original string response in case of success.